test/test_torch.py

```diff
@@ -13538,7 +13549,7 @@ def test_reverse_binary_ops_multiple_device(self, devices):
         self.assertEqual(2 + torch.tensor(3), 2 + torch.tensor(3).to(devices[1]))    # __radd__
         self.assertEqual(2 - torch.tensor(3), 2 - torch.tensor(3).to(devices[1]))    # __rsub__
         self.assertEqual(2 * torch.tensor(3), 2 * torch.tensor(3).to(devices[1]))    # __rmul__
-        self.assertEqual(2 / torch.tensor(3), 2 / torch.tensor(3).to(devices[1]))    # __rtruediv__
+        self.assertEqual(2 / torch.tensor(3.), 2 / torch.tensor(3.).to(devices[1]))  # __rtruediv__
```

Comment:
for these that you changed from ints to floats, can you just check that there are places here that test floor_div on ints.

Reply:
Yep (see the line just below this one, for example).